samples/nrf5340/extxip_smp_svr/README.rst

```diff
@@ -0,0 +1,184 @@
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates how to split an application that partially resides on internal flash and Quad Serial Peripheral Interface (QSPI) flash by using the Simple Management Protocol (SMP) server.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample requires a connection to a computer with :file:`mcumgr` command-line tool's or a compatible Mcumgr client tool.
+
+Overview
+********
+
+SMP is a basic transfer protocol used by the MCUmgr management library.
+This sample shows how to divide an application into internal and external parts using the SMP server.
+The internal part operates from the SoC's built-in storage, and the external part runs directly from external memory, using the XIP capability through a QSPI peripheral.
+
+For more information about MCUmgr and SMP, see :ref:`device_mgmt`.
+
+The sample enables the splitting by:
+
+* Enabling the split XIP image feature through the :kconfig:option:`CONFIG_XIP_SPLIT_IMAGE` Kconfig option.
+  This Kconfig option sets a build-system level support for image division and adapts the image-signing support for MCUboot.
+* Using the project's CMake file which describes relocation of certain libraries or objects to an external XIP area.
+* Using a linker script that describes the external QSPI area.
+  The code for execution from the external memory should link to it.
+* Using the static partition manager description for appropriate memory mapping.
+
+For details on SMP features implemented by the sample, see the Zephyr-RTOS :zephyr:code-sample:`smp-svr`.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf5340/extxip_smp_svr`
+.. include:: /includes/build_and_run.txt
+
+The |NCS| build system generates all essential binaries, including the application for internal and external QSPI flash, the networking stack, and the MCUboot.
+The build process involves signing all the application binaries except for MCUboot which does not require signing in this configuration.
+
+Flashing the sample image
+=========================
+
+To upload MCUboot and a bundle of images to the nRF5340 SoC , use the ``west flash`` command.
+
+For Thingy:53, however, the QSPI memory requires a dedicated configuration file for flashing.
+In such a case, run the following command instead:
+
+.. code-block:: console
+
+    nrfjprog -f NRF53 --coprocessor CP_NETWORK --sectorerase --program hci_ipc/zephyr/merged_CPUNET.hex --verify
+    nrfjprog -f NRF53 --sectorerase --program mcuboot/zephyr/zephyr.hex --verify
+    nrfjprog -f NRF53 --sectorerase --program zephyr/internal_flash_signed.hex --verify
+    nrfjprog -f NRF53 --qspisectorerase --program zephyr/qspi_flash_signed.hex --qspiini <path_to_sample>/Qspi_thingy53.ini --verify
+    nrfjprog -f NRF53 --reset
+
+Sample image: Hello world!
+--------------------------
+
+The ``smp_svr`` app is now ready to run.
```

Comment:
```suggestion
The ``smp_svr`` app is now ready to run.
You can test it by using a "Hello world" sample image.
```

Reply:
> You can test it by using a "Hello world" sample image.

It can be tested using mcumgr. the "Hello word" phrase was only  in title of chapter before.
I would kept text as is.